File: treatment/src/test/java/com/hartwig/actin/treatment/trial/config/TestTrialDefinitionConfigFactory.java
package com.hartwig.actin.treatment.trial.config;

import org.apache.logging.log4j.util.Strings;
import org.jetbrains.annotations.NotNull;

public final class TestTrialDefinitionConfigFactory {

    @NotNull
    public static ImmutableTrialDefinitionConfig.Builder builder() {
        return ImmutableTrialDefinitionConfig.builder().trialId(Strings.EMPTY).open(false).acronym(Strings.EMPTY).title(Strings.EMPTY);
    }
}

File: treatment/src/test/kotlin/com/hartwig/actin/treatment/trial/config/TestTrialDefinitionConfigFactory.kt
package com.hartwig.actin.treatment.trial.config

import org.apache.logging.log4j.util.Strings

object TestTrialDefinitionConfigFactory {
    val MINIMAL = TrialDefinitionConfig(
        trialId = Strings.EMPTY,
        open = null,
        acronym = Strings.EMPTY,
        title = Strings.EMPTY
    )
}